processing/src/test/java/org/apache/druid/query/aggregation/AggregatorFactoryTest.java

```diff
@@ -307,7 +307,7 @@ public void testWithName()
         // string aggregators
         new StringFirstAggregatorFactory("col", "col", null, 1024),
         new StringLastAggregatorFactory("col", "col", null, 1024),
-        new StringAnyAggregatorFactory("col", "col", 1024),
+        new StringAnyAggregatorFactory("col", "col", 1024, true),
```

Comment:
Just for sake of completeness can we add a test with false as the 3rd paramter as well ?